packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -449,7 +491,24 @@ class Scrollable extends Component {
 	}
 
 	render () {
-		const {focusableScrollbar, childRenderer, ...rest} = this.props;
+		const
+			{
+				childRenderer,
+				focusableScrollbar,
+				horizontalScrollbarNextButtonAriaLabel,
+				horizontalScrollbarPreviousButtonAriaLabel,
+				verticalScrollbarNextButtonAriaLabel,
+				verticalScrollbarPreviousButtonAriaLabel,
+				...rest
+			} = this.props,
+			horizontalScrollbarA11yProps = {
```

Comment:
Let's include these inline within the JSX. In general, we try to keep all props defined within the JSX. There are some cases in which is makes sense to bundle them separately (e.g. there are props that are defined externally such as in some HOCs or props are conditionally passed) but this case causes an extra object allocation and affects readability by adding another property bag to be mentally parsed.